os-eap-probes/added/probes/probe/dmr.py

```diff
@@ -49,6 +49,8 @@ def __readConfig(self):
         self.port = 9990 + int(os.getenv('PORT_OFFSET', 0))
         self.user = os.getenv('ADMIN_USERNAME')
         self.password = os.getenv('ADMIN_PASSWORD')
+        if self.password != "" and self.user == "":
```

Comment:
this only works if ADMIN_USERNAME is set to an empty string, if it is not set at all, it will not be set to "eapadmin", as it should..